### ISSUE TITLE
[prod] Fix SMTP authentication

### ISSUE THREAD

**Issue by jakubgs** [dap-ps/infra-dapps#9]
Apparently emails are not being sent.

**Comment by jakubgs** [comment]
I've tested with python and it see,s the credentials indeed do not work:
```ipython
In [1]: import smtplib, ssl, os

In [3]: username = os.environ["SMTP_USER"]

In [4]: password = os.environ["SMTP_PASS"]

In [5]: context = ssl.create_default_context()

In [6]: with smtplib.SMTP_SSL("email-smtp.us-east-1.amazonaws.com", 465, context) as server:
   ...:     server.login(username, password)
   ...:     server.send_message("Test Message", "dapps-approvals@status.im", "dapps-approvals@status.im")

SMTPAuthenticationError: (535, b'Authentication Credentials Invalid')
```

**Comment by jakubgs** [comment]
Ok, figured out what was the issue. I was extracting the password from the [`iam_access_key`](https://www.terraform.io/docs/providers/aws/r/iam_access_key.html) resource using [`encrypted_secret`](https://www.terraform.io/docs/providers/aws/r/iam_access_key.html#encrypted_secret) and derypting it with `base64 --decode | gpg` rather than using [`ses_smtp_password`](https://www.terraform.io/docs/providers/aws/r/iam_access_key.html#ses_smtp_password) which does not need decryption. Fixed in 023d2bea60c9592dfdd185cde581966e67cea7f3.

**Comment by jakubgs** [comment]
Also added a utility for testing SES SMTP:
https://github.com/status-im/infra-utils/blob/master/smtp/ses_email.py

Seems to work:
```bash
~/work/infra-utils master*
 lilim > export SMTP_USER=$(pass cloud/AWS/ses/smtp-access-key) 

~/work/infra-utils master*
 lilim > export SMTP_PASS=$(pass cloud/AWS/ses/smtp-password)  

~/work/infra-utils master*
 lilim > ./smtp/ses_email.py                                   
mail successfully sent
```

**Comment by jakubgs** [comment]
I also had to fix from which credentials the SMTP auth comes from in `terraform.tfvars` file: ed59190434d0bacc935b1d6cd10e4fe40b95bc90

Deployed fixed credentials to both `dev` and `prod`. Now I have to try to submit a Dapp.

**Comment by jakubgs** [comment]
So before we were getting this from app backend:
```
[ERROR]-[Base-Email]: Email service verification failed due to Error: Invalid login: 535 Authentication Credentials Invalid
```
But now it's this:
```
[Base-Email]: Email service verification failed due to Error: 139837730322304:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:252:
```
This might require changes to how the JS sends emails.

**Comment by jakubgs** [comment]
According to this:
https://github.com/nodemailer/nodemailer/issues/165
Adding options like explicit tls cipher and disabling secure connection could help:
```javascript
var smtpTransport = nodemailer.createTransport("SMTP", {
    host: "***.***.***.***",
    secureConnection: true,
    port: 465,
    auth: {
            user: "****",
            pass: "****"
    },
    tls: {
            ciphers:'SSLv3'
    }
});
```

**Comment by jakubgs** [comment]
I tried locally with a script and it delivers without issues:
https://github.com/status-im/infra-utils/blob/master/smtp/ses_email.js
But they end up in SPAM, so that's another issue:
![spam](https://user-images.githubusercontent.com/2212681/62553916-32b95280-b83e-11e9-96ed-889414518f89.png)

According to AWS documentation:
https://docs.aws.amazon.com/ses/latest/DeveloperGuide/received-email-problems.html

>Some email clients display the "via" domain when the sender's domain does not match the domain that the email was actually sent from (in this case, amazonses.com).

They link to this:
https://support.google.com/mail/answer/1311182?hl=en;

>The domain it was sent from doesn't match the domain in the "From:" address. For example, you got an email from john.smith@gmail.com, but it could've been sent through a social networking site and not Gmail.

They suggest adding an SPF record(we already have DKIM):
>* If you send messages using a bulk mailing vendor or a third party affiliates, check out Gmail's Bulk Senders Guidelines, including: Publish an SPF record that includes the IPs of the vendor or affiliates which send your messages.

**Comment by jakubgs** [comment]
Okay, i see the issue, it was me using `dapps-approvals@status.im` instead of `dapps-approvals@dap.sp` as a sender: https://github.com/status-im/infra-utils/commit/42603ec5f074c7e1f801587a6143d1e9f85ecfb1

**Comment by jakubgs** [comment]
And fixed here for EB env vars: 649e4cb6c740d760320fabf8cb3b8260e6604324

**Comment by jakubgs** [comment]
Aaaaand it works:
```
[INFO]-[DApps-Metadata-Controller]: A dapp metadata with hash [QmVUyB8JKchLSK4CX7tYEuFznsiNXYzz7YnE8mrWZq8PVG] has been uploaded successfully
[INFO]-[Base-Email]: Email with subject Uploaded DApp Metadata. Hash - QmVUyB8JKchLSK4CX7tYEuFznsiNXYzz7YnE8mrWZq8PVG was delivered successfully
```
![mother of god](https://user-images.githubusercontent.com/2212681/62555279-a65c5f00-b840-11e9-951a-648d31187202.jpg)
